packages/block-editor/src/components/inspector-controls/groups.js

```diff
@@ -13,10 +13,14 @@ const InspectorControlsDimensions = createSlotFill(
 const InspectorControlsTypography = createSlotFill(
 	'InspectorControlsTypography'
 );
+const InspectorControlsBlockInformation = createSlotFill(
+	'InspectorControlsBlockInformation'
+);
 
 const groups = {
 	default: InspectorControlsDefault,
 	advanced: InspectorControlsAdvanced,
+	blockInformation: InspectorControlsBlockInformation,
```

Comment:
just "info" ?

Reply:
Sounds good to me. ✅ Done https://github.com/WordPress/gutenberg/pull/45437/commits/473bf49593492d25b9a3c3185ef4c8fff20e18c6.